src/Profile/InfoBoxConfig.cpp

```diff
@@ -127,6 +127,10 @@ Profile::Load(const ProfileMap &map, InfoBoxSettings &settings)
     break;
   }
 
+  map.Get(ProfileKeys::InfoBoxTitleScale, settings.scale_title_font);
+  if ((settings.scale_title_font < 80) || (settings.scale_title_font > 150))
+    settings.scale_title_font = 100;
```

Comment:
What's the point of this line?
Did you want to clamp?

Reply:
settings.scale_title_font comes from the active PRF file, which may hold any value.
If someone edits the PRF and sets the value to 1000, this will make sure settings.scale_title_font has a reasonable value.

Replace 80 by 50 to make it consistent throughout the changes.